runtime/frequency-rococo/src/lib.rs

```diff
@@ -81,14 +81,21 @@ pub type SignedExtra = (
 	pallet_msa::CheckFreeExtrinsicUse<Runtime>,
 );
 
-/// Extrinsic type that has already been checked.
-pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, Call, SignedExtra>;
 /// A Block signed with a Justification
 pub type SignedBlock = generic::SignedBlock<Block>;
 
 /// BlockId type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
 
+/// Block type as expected by this runtime.
```

Comment:
I moved these to match frequency/src/lib.rs for easier comparison when configs are changed.